CONTRIBUTING.md

```diff
@@ -149,17 +149,27 @@ Releases are cut with `git` tags and a successful Travis build results in new bi
 
 Credentials for the `openfaas` and/or `functions` Docker Hub and quay.io accounts are coordinated by the project lead using.
 
+## Governance
+
+OpenFaaS is an independent open-source project which was created by the Project Lead Alex Ellis in 2016. OpenFaaS is now being built by Alex, a number of volunteer teams, and a wider community of open-source developers.
+
+OpenFaaS Ltd (company no. 11076587) sponsors the development and maintenance of OpenFaaS. OpenFaaS Ltd provides professional services, consultation and support.
+
+OpenFaaS &reg; is a registered trademark in England and Wales.
+
 ### How do I become a maintainer?
 
-In the OpenFaaS community there are three levels of maintainership:
+In the OpenFaaS community there are four levels of structure or maintainership:
 
-* Core Contributors
-* GitHub Organisation Members
+* Core Team (GitHub org)
+* Members Team (GitHub org)
 * Those with Derek access
 
-#### Who are the Core Contributors?
+The rest of the community.
```

Comment:
should this be a bullet point? you mention "four levels" but then have only 3 points, looks a little odd at first.

Reply:
yep